benchmark/benchmark_runner.cpp

```diff
@@ -145,20 +158,50 @@ double benchmark_runner::calculate_driver_bench_score() {
     return (calculate_read_bench_score() + calculate_write_bench_score()) / 2.0;
 }
 
-void benchmark_runner::print_scores() {
+void benchmark_runner::write_scores(
+    const std::chrono::time_point<std::chrono::system_clock> start_time) {
     double read = -1;
     double write = -1;
+    const auto end_time = std::chrono::system_clock::now();
+
+    using namespace bsoncxx;
+    using builder::basic::sub_document;
+
+    auto doc = builder::basic::document{};
+    doc.append(kvp("info", [](sub_document subdoc) {
+        subdoc.append(kvp("test_name", "C++ microbenchmarks"));
```

Comment:
```suggestion
        subdoc.append(kvp("test_name", "C++ Driver microbenchmarks"));
```
May be worth specifying that this is the C++ _Driver_ that is being benchmarked.